Mathlib/Control/Bitraversable/Instances.lean

```diff
@@ -46,7 +46,10 @@ instance : Bitraversable Prod where bitraverse := @Prod.bitraverse
 
 instance : LawfulBitraversable Prod := by
   constructor <;> intros <;> casesm _ × _ <;>
-    simp [bitraverse, Prod.bitraverse, functor_norm] <;> rfl
+    simp only [bitraverse, Prod.bitraverse, Function.comp_apply, Functor.Comp.map_mk,
```

Comment:
Here also, I'm inclined to make an exception for `simp [...] <;> rfl`. The problem with non-terminal simps is that they are hard to maintain. But `simp; rfl` is an exception to the general rule, I think.
And the squeeze is obscuring the proof, I think...